lib/json-schema/validator.rb

```diff
@@ -18,15 +18,15 @@ module JSON
   class Validator
 
     @@schemas = {}
-    @@cache_schemas = false
+    @@cache_schemas = true
```

Comment:
The default behaviour previously was to cache schemas, even though the `@@cache_schemas` variable was false